app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -67,18 +71,18 @@
     private static final String NEWLINE = "\n";
     private static final Region region = Region.getRegion(Regions.US_EAST_1);
 
-    private String supportEmail;
+    private String supportEmail = "support@sagebridge.org";
```

Comment:
We should move this value to Spring, so we can still change it in config instead of code if we need it.
<bean id="numAsyncWorkerThreads" class="java.lang.String">
  <constructor-arg value="support@sagebridge.org" />
</bean>

Also, the previous value was "Bridge (Sage Bionetworks) &lt;support@sagebridge.org&gt;". Was this change intentional?